cmd/zpool_influxdb/README.md

```diff
@@ -237,7 +237,7 @@ be restarted to read the config-directory files.
 # # Read metrics from zpool_influxdb
 [[inputs.execd]]
 #   ## default installation location for zpool_influxdb command
-  command = ["/usr/bin/zpool_influxdb", "--execd"]
+  command = ["/usr/libexec/zpool_influxdb", "--execd"]
```

Comment:
question: since there already exists an {prefix}/libexec/zfs should we place it there, or is {prefix}/libexec better?

Reply:
It means less code to maintain; no need for any install code, as autotools already support installing `libexec_PROGRAMS`; so that's why I used that.
Also when I look some random Ubuntu installation, there's a whole bunch of executables directly in the `/usr/libexec` dir - so for things that can actually be executed straight away and aren't just some support binaries needed to be executed as a part of some bigger machinery, I think using the `libexec` dir directly could make some sense.

Reply:
ok